clean_files.txt

```diff
@@ -48,10 +48,11 @@ plugins/available/basher.plugin.bash
 #
 completion/available/cargo.completion.bash
 completion/available/knife.completion.bash
+completion/available/notify-send.completion.bash
+completion/available/notify-send.completion.bash
```

Comment:
Looks like you have this line twice.

And `vault.completion.bash` seems to be gone...

Reply:
Done

Reply:
What happened to the `vault.completion.bash` file entry?

Reply:
Removed

Reply:
why did you remove it @tbhaxor ? bring it back and its pretty ready